tracer/src/Datadog.Trace/ServiceFabric/ServiceRemotingHelpers.cs

```diff
@@ -90,7 +90,11 @@ public static void GetMessageHeaders(EventArgs? eventArgs, out IServiceRemotingR
 
         public static string GetSpanName(string spanKind)
         {
+#if NET6_0_OR_GREATER
+            return string.Create(null, stackalloc char[128], $"{SpanNamePrefix}.{spanKind}");
```

Comment:
_Technically_ there's a potential issue here if `spanKind.Length > 128 - SpanNamePrefix.Length`. It's very unlikely, and this is `internal` and we control the `spanKind` types in this callpath. But it still gives me slight eye twitch! 😄